index.html

```diff
@@ -110,8 +110,8 @@ <h1>Welcome to Steven's CSD120 Website!</h1>
           </table>
         </div>
         <h2>My favorite things.</h2>
-        <div class="favorites">
-          <input type="radio" id="tab1" name="mythings" /><label for="tab1"
+        <div class="favourites">
```

Comment:
I changed the spelling so my tester can detect your styles automatically... no marks lost